mozilla_django_oidc/views.py

```diff
@@ -123,7 +123,7 @@ def redirect_url(self):
         """Return the logout url defined in settings."""
         return import_from_settings('LOGOUT_REDIRECT_URL', '/')
 
-    def dispatch(self, request, *args, **kwargs):
+    def post(self, request):
```

Comment:
Tests test that automatically. As long as the view doesn't have a `csrf_exempt` decorator it should be using CSRF.